### PR TITLE
extend the resolution's limitation from 4k to 8k

### PR DESCRIPTION
**Pull request by dyang23**
Tracked-On: OAM-96722
Signed-off-by: Yang, Dong <dong.yang@intel.com>

### CODE REVIEW

**Review by dyang23 — COMMENTED** [review]
@TianmiChen @zhe.zhu@intel.com pleaser review

**Review by TianmiChen — APPROVED** [review]

File: omx_components/src/mfx_omx_vdec_component.cpp [hunk 1]
```diff
@@ -2218,7 +2218,7 @@ mfxStatus MfxOmxVdecComponent::InitCodec(void)
         }
         if (MFX_ERR_NONE == mfx_res)
         {
-            if (m_MfxVideoParams.mfx.FrameInfo.Width > 4096 || m_MfxVideoParams.mfx.FrameInfo.Height > 4096)
+            if (m_MfxVideoParams.mfx.FrameInfo.Width > 8192 || m_MfxVideoParams.mfx.FrameInfo.Height > 8192)
```
Comment (TianmiChen) [inline omx_components/src/mfx_omx_vdec_component.cpp:2221]: Can we exclude VP8 hw decoder? VP8 can only support 4K.

File: omx_components/src/mfx_omx_vdec_component.cpp [hunk 2]
```diff
@@ -2542,7 +2542,7 @@ mfxStatus MfxOmxVdecComponent::ReinitCodec(void)
     }
     if (MFX_ERR_NONE == mfx_res)
     {
-        if (newVideoParams.mfx.FrameInfo.Width > 4096 || newVideoParams.mfx.FrameInfo.Height > 4096)
+        if (newVideoParams.mfx.FrameInfo.Width > 8192 || newVideoParams.mfx.FrameInfo.Height > 8192)
```
Comment (TianmiChen) [inline omx_components/src/mfx_omx_vdec_component.cpp:2545]: As above.

Comment (dyang23) [reply omx_components/src/mfx_omx_vdec_component.cpp:2545]: let's use the profiles as platform codec's differenciation.

**Review by LukeZheZhu — APPROVED** [review]

**Review by LukeZheZhu — APPROVED** [review]

**Review by LukeZheZhu — APPROVED** [review]

**Review by TianmiChen — APPROVED** [review]

**Pull request merged**